es_client.py

```diff
@@ -42,6 +62,29 @@ def tweet_to_estweet(t, stored_at, version):
     return ans
 
 
+def tweepy_to_dict(tweet, *, stored_at=None, version=None):
```

Comment:
```suggestion
def tweepy_to_dict(tweet, **kwargs):
```
I'm assuming `*` is a variadic argument and we are discarding all other params except tweet, stored_at, version. In this case, use **kwargs instead and check if 'stored_at' and 'version' are in kwargs. An alternative implementation would be to change the `*` to `*args` to improve readability.

Reply:
Refactored in [4b3418b](https://github.com/fund3/sentiment-dashboard/pull/6/commits/4b3418b639490ec345ee9ed21e6a995713c2e4b7).